src/control/security/config.go

```diff
@@ -61,7 +60,6 @@ func (tc *TransportConfig) String() string {
 // component. ServerName is only needed if the config is being used as a
 // transport credential for a gRPC tls client.
 type CertificateConfig struct {
-	ServerName      string           `yaml:"server_name,omitempty"`
```

Comment:
The example configs and documentation will probably need to be updated as part of this work.

Reply:
I dont believe any of the sample configs have that value in them anymore. I will check to make sure though.